src/main/java/ro/societateahermes/backendservice/service/ActivityServiceInterface.java

```diff
@@ -0,0 +1,18 @@
+package ro.societateahermes.backendservice.service;
+
+import ro.societateahermes.backendservice.entities.Activity;
+import ro.societateahermes.backendservice.entities.DTO.ActivityDTO;
+import ro.societateahermes.backendservice.entities.DTO.UserDTO;
+import ro.societateahermes.backendservice.entities.User;
+
+import java.util.List;
+
+public interface ActivityServiceInterface {
+    void save(ActivityDTO activity);
+
+    List<Activity> getAllActivities();
```

Comment:
trebuie sa trimita catre controller o lista de dto sau sa convertesti din entitate in dto in controller